packages/common/src/components/NbConda.tsx

```diff
@@ -355,6 +364,60 @@ export class NbConda extends React.Component<ICondaEnvProps, ICondaEnvState> {
     }
   }
 
+  async handleSolveEnvironment(): Promise<void> {
+    let toastId: React.ReactText;
+    const model = this.props.model;
+    const { subdir } = await model.subdir();
+
+    const create = async (name: string, explicitList: string) => {
+      toastId = await INotification.inProgress(`Creating environment ${name}`);
+      try {
+        await model.createFromExplicitList(name, explicitList);
```

Comment:
You can reuse `import`

```suggestion
        await model.import(name, explicitList);
```